README.md

```diff
@@ -88,22 +78,57 @@ $ python3 -m pip install semgrep
 $ docker run --rm -v "${PWD}:/src" returntocorp/semgrep semgrep
 ```
 
-Once installed, Semgrep can run with single rules or entire rulesets. Visit [Docs > Running rules](https://semgrep.dev/docs/running-rules/) to learn more or try the following:
+2.  Run `semgrep login` to get the login URL. Open the login URL in the browser and login [Optional, but recommended].
```

Comment:
You don't need to open the login url. `semgrep login` should be sufficient and should open the browser automatically

Reply:
It never opened the browser for me. I thought it was not implemented.